### ISSUE TITLE
Next release planning

### ISSUE THREAD

**Issue by henryiii** [wntrblm/nox#866]
No promises here, but this my personal wishlist for the next release:

Related series of ideas:

- #631 - this is the key building block for making one thing depend on other things

Add some helpers to the `nox.project` namespace for common tasks that have PEPs behind them:

- #876
- #877 (followup #884)

PEP 723 runner:
- #878 - Easier runs via PEP 723
- #881 - This would also users to share code, and even not have to install nox. Also extend using Python packages, like using click or rich inside a session.

Fixes:
- #829
- #871
- #872
- #874
- #913
- #908
- #901 
- #900 
- #891 
- #889
- #887
- #880 (and fix in #921)
- #917
- #916
- #918
- #922

Cleanup (with fixes):
- #889 
- #890
- #893
- #894
- #910
- #912
- #882 

Tests
- #895 
- #894 
- #893

Longer term, I have ideas about the CLI that might make sense after #167.

Postponed:

- #167 - allow `@nox.env` and then defining sessions for it. Would probably solve a lot of things at once, like custom venv locations and lock file support.
- #740 - add a "build" job and way to share wheels - could be just an example using the above one or two items.
- #814 - this would also allow PEP 721's requires-python setting to be used.
- Verify 3.13t works well

**Comment by theacodes** [comment]
Alright, I cut the release and everything seems to have published. Thank you, @henryiii. :)
